.coderabbit.yaml

```
@@ -13,8 +13,8 @@ reviews:
   collapse_walkthrough: false
   path_filters:
     - '**'
-    - 'backend/ent/**'
-    - '!backend/ent/schema/**'
+    - '!backend/ent/**'
+    -  'backend/ent/schema/**'
```

yamllint

[warning] too many spaces after hyphen

(hyphens)